packages/cli/lib/services/init.service.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'node:fs';
+import path, { dirname } from 'node:path';
+import chalk from 'chalk';
+import { printDebug } from '../utils.js';
+import { nangoConfigFile } from '@nangohq/nango-yaml';
+import { generate } from '../cli.js';
+import { fileURLToPath } from 'node:url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+/**
+ * Init
+ * If we're not currently in the nango-integrations directory create one
+ * and create an example nango.yaml file
+ */
+export function init({ absolutePath, debug = false }: { absolutePath: string; debug?: boolean }): boolean {
+    const yamlData = fs.readFileSync(path.resolve(__dirname, `../templates/${nangoConfigFile}`), 'utf8');
+    const envData = fs.readFileSync(path.resolve(__dirname, '../templates/env'), 'utf8');
+    const gitIgnoreData = fs.readFileSync(path.resolve(__dirname, '../templates/gitignore'), 'utf8');
```

Comment:
is this really necessary considering the file is super small?
also it should not be empty but should contains ```dist
.env```

Reply:
I was trying to move these all to file copies just so they were a little easier to modify and more consistent to look at, but I can revert.

Reply:
Fixed in 3986c3553

Reply:
I was just referring to the gitignore, I was perfectly okay with the rest au contraire 👌🏻 sorry for the confusion
I need to be more careful when suggesting, but you don't have to revert when you have a suggestion ☺️

Reply:
😄 it was trivial and I was apathetic, so figured why not.